go.mod

```diff
@@ -0,0 +1,28 @@
+module github.com/edgexfoundry/edgex-go
+
+require (
```

Comment:
Confirming that this is exactly the set of `require`s that I, too, got generated separately, when I created a `go module` out of the edgex-go repo 👍